### PR TITLE
Analysis update: Add MB workflow

### PR DESCRIPTION
**Pull request by zzgeng**
#### What scientific question is your analysis addressing?
Analysis update: Add MB workflow

#### What was your approach?
Analysis update: Add MB workflow

#### Which areas should receive a particularly close look?
MB workflow plot

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
Yes

#### Reproducibility Checklist

- [ ] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [ ] This analysis has been added to continuous integration.

#### Documentation Checklist

- [ ] This analysis module has a `README` and it is up to date.
- [ ] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [ ] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/molecular-subtyping-MB/plot/MB_subtyping.png
Comment (JosephDybas) [inline analyses/molecular-subtyping-MB/plot/MB_subtyping.png:1]: The figure looks good but I think the flow is a little confusing. Two things that I would take another look at.
1)There are two arrows coming out of the "clinical information" node that each go to another node that states "Samples without RNA-seq". Should the rightmost arrow be "samples without RNA-seq AND without methylation data" and the middle arrow be "samples without RNA-seq AND with methylation data"? If so, maybe make one arrow to a "samples without RNA-seq" node and then two arrows off that one going to "With methylation data" and one going to "without methylation data". Then to the classification scheme or to the "to be classified" node.
2) There are no descriptors of how the MB classifiers call the different subtypes (SHH, Group3, Gropu4, WNT, MYO). Is this intended?

**Review by JosephDybas — APPROVED** [review]

**Review by jharenza — CHANGES_REQUESTED** [review]
I tend to agree with @JosephDybas on this, that the flow is confusing with multiple arrows. 

For one thing, we are only using medulloPackage here for RNAseq classification. The other may still be run here, but we are no longer using it or consensus in OPC as we were in OpenPBTA, so we need to remove MM2S. It may be easier to read if the RNA-Seq and methyl flows were completely separated. I think it is OK to put the 4 subtypes from RNAseq separate of the 5 subtypes from methyl. (MB, MYO is not a subtype called from RNAseq).

**Review by jharenza — APPROVED** [review]

File: analyses/molecular-subtyping-MB/plot/MB_subtyping.png
Comment (jharenza) [reply analyses/molecular-subtyping-MB/plot/MB_subtyping.png:1]: >There are no descriptors of how the MB classifiers call the different subtypes (SHH, Group3, Gropu4, WNT, MYO). Is this intended?

@JosephDybas for the purposes of this module, we refer to our publication https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008263 - beyond the scope for the diagram but @zzgeng if this pub is not in the readme, can you please add before merge? Thanks!

Comment (JosephDybas) [reply analyses/molecular-subtyping-MB/plot/MB_subtyping.png:1]: I agree that the reference should be added to the README (I didn't see it but perhaps I missed it). Also, you could add something like "Rathi et. al. PLoS Comp Bio 2020" above the horizontal line connecting all the vertical arrows coming out of the medulloPackage node on the figure, or with an asterisks in the legend somehow, if you like. But just a suggestion; it's not necessary if it's in the readme. Thanks!

**Pull request merged**